src/main/java/org/openstreetmap/atlas/geography/atlas/items/Relation.java

```diff
@@ -110,8 +117,7 @@ public JsonObject asGeoJsonGeometry()
             {
                 final String message = String.format("%s - %s",
                         exception.getClass().getSimpleName(), exception.getMessage());
-                properties.addProperty("exception", message);
-                logger.warn("Unable to recreate multipolygon for relation {}.", getIdentifier(),
+                logger.error("Unable to recreate multipolygon for relation {}.", getIdentifier(),
```

Comment:
Sonar has also pointed this out, but you are missing a pair of squiggly braces in the message template.